server/test/login.test.js

```diff
@@ -0,0 +1,60 @@
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const { server } = require("../app.js");
+
+chai.should();
+chai.use(chaiHttp);
+
+const loginDetailInfo={
+  'email':'panwar.amita@gmail.com',
+  'password':'123456'
+}
+const loginDetailInvlidEmail={
+  'username':'hanry',
+  'email':'panagmail.com',
+  'password':'amita@123'
+}
+const loginDetailInvalidPassword={
+  'username':'123',
+  'email':'panag@mail.com',
+  'password':''
+}
+describe("/POST /auth/login", () => {
+  it("it should return 200 and message", (done) => {
+    chai
+      .request(server)
+      .post('/auth/login/')
+      .send(loginDetailInfo)
+      .end((err, res) => {
+        const cookie=res.headers['set-cookie']; 
+        res.should.have.status(200);
+         cookie.should.not.be.empty;
```

Comment:
Nit: Please fix indentation - you can set up a code formatter like Prettier

Reply:
Done the formatting.